modules/apps/hooks/testutils/testing_hooks.go

```diff
@@ -0,0 +1,54 @@
+package testutils
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	capabilitytypes "github.com/cosmos/cosmos-sdk/x/capability/types"
+	ibchooks "github.com/cosmos/ibc-go/v5/modules/apps/hooks"
+	channeltypes "github.com/cosmos/ibc-go/v5/modules/core/04-channel/types"
+	ibcexported "github.com/cosmos/ibc-go/v5/modules/core/exported"
+)
+
+var _ ibchooks.IBCAppHooks = TestRecvOverrides{}
+var _ ibchooks.IBCAppHooks = TestRecvBeforeAfterHooks{}
+
+type Status struct {
+	OverrideRan bool
+	BeforeRan   bool
+	AfterRan    bool
+}
+
+// Recv
+type TestRecvOverrides struct{ Status *Status }
```

Comment:
you can... but then you have to add a `NewTestRecvOverrideHooks`, which I find less clean. Since it's only used in testing, I don't see the harm in having it be exported.

Reply:
Don't feel strongly either way, though. Happy to change it if you do